src/main/java/homework/storage/Storage.java

```diff
@@ -0,0 +1,9 @@
+package homework.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> dataBase = new HashMap<>();
```

Comment:
Better to have a map of fruits and their quantities, e.g.
     public static final Map<Fruit, Integer> dataBase = new HashMap<>();